nano/node/active_transactions.cpp

```diff
@@ -638,4 +739,27 @@ std::unique_ptr<seq_con_info_component> collect_seq_con_info (active_transaction
 	composite->add_component (std::make_unique<seq_con_info_leaf> (seq_con_info{ "confirmed", confirmed_count, sizeof (decltype (active_transactions.confirmed)::value_type) }));
 	return composite;
 }
+transaction_counter::transaction_counter () :
+trend_last (std::chrono::steady_clock::now ()),
+counter (0),
+rate (0)
```

Comment:
The order in the construct member initialization list is not the same as the order the members are declared in the class. Could just add the members to the class definition and remove the need for this constructor entirely, get 2 birds stoned at once :).